server/shared/sse/index.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+const Channel = require('./channel');
+const cuid = require('cuid');
+const { EventEmitter } = require('events');
+const nocache = require('nocache')();
+
+const reEOL = /\r?\n/g;
+const SSE_TIMEOUT_MARGIN = 0.10;
+const SSE_DEFAULT_TIMEOUT = 60000; /* ms */
+const SSE_HEADERS = {
+  connection: 'keep-alive',
+  'transfer-encoding': 'identity'
```

Comment:
If you want this to work behind nginx you need to either disable `proxy_buffering` in nginx configuration (which disables it for everything) or add additional headers to tell nginx to disable it just for our sse connection:
```js
  'Cache-Control': 'no-cache',
  'X-Accel-Buffering': 'no'
```